tx/textual/internal/testdata/any.json

```diff
@@ -0,0 +1,72 @@
+[
+    {
+        "url": "type.googleapis.com/Foo",
+        "msg": {},
+        "screens": [
+            {"text": "Object: type.googleapis.com/Foo"},
+            {"text": "Foo object", "indent": 1}
+        ]
+    },
+    {
+        "url": "type.googleapis.com/Foo",
+        "msg": {
+            "full_name": "testing"
+        },
+        "screens": [
+            {"text": "Object: type.googleapis.com/Foo"},
+            {"text": "Foo object", "indent": 1},
+            {"text": "Full name: testing", "indent": 2}
+        ]
+    },
+    {
+        "url": "type.googleapis.com/google.protobuf.Timestamp",
+        "msg": {
+            "seconds": 1136214245
+        },
```

Comment:
Here the url/msg combo is just representing an Any in a custom way. Why not use protojson's representation?

```json
  "proto": {
    "@type": "type.googleapis.com/google.protobuf.Timestamp",
    "seconds": 1136214245
  }
```

We also get consistency with the other test cases with the "proto" field in our JSON test case.

Reply:
Done - I didn't notice `protojson`, and that works quite well! I nearly managed to erase the controversial `type.googleapis.com` prefix from the test cases, since I switched to just comparing the internal messages, but it still shows up in the nested `Any` cases.